src/Indigo/Contracts/HomebaseLite/Types.hs

```diff
@@ -0,0 +1,67 @@
+-- SPDX-FileCopyrightText: 2022 Tezos Commons
+-- SPDX-License-Identifier: LicenseRef-MIT-TC
+
+{-# OPTIONS -Wno-orphans #-}
+
+module Indigo.Contracts.HomebaseLite.Types
+  ( Parameter(..)
+  , Storage(..)
+  , Configuration(..)
+  , URI(..)
+  ) where
+
+import Indigo
+
+data Parameter
+  = Set_admin KeyHash
+  | Accept_admin ()
+  | Add_maintainers [KeyHash]
+  | Remove_maintainers [KeyHash]
+  | Configure Configuration
+  | Propose (URI, [MText])
+  | Vote (URI, Natural)
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue)
+
+[entrypointDoc| Parameter plain |]
+
+data Configuration = Configuration
+  { cExpireTime :: Natural
+  , cVoteDelay :: Natural
```

Comment:
What do you think about making a newtype for `Seconds` for clarity?

BTW `nat`s can't be subtracted from `timestamp`s in Michelson, so a conversion to `int` will be needed, however I still think it makes sense to pick `nat`s here.